### DISCUSSION TITLE
How about a new icon

### DISCUSSION THREAD

**Discussion by Tunsinasro** [spacecowboy/Feeder#169] [Ideas]
I have a proposition regarding the app icon and believe a new design could enhance the overall aesthetic

**Comment by spacecowboy** [comment]
You could show your icon suggestion here if you want

**Comment by Tunsinasro** [comment]
How about this one, is AI generated
![Screenshot_2023-12-27-10-30-16-263_com android chrome-edit](https://github.com/spacecowboy/Feeder/assets/147873384/d2b10c21-18e8-45b4-992c-db80603bad65)

**Comment by Tunsinasro** [comment]
Or this one, i changed the main color to orange
![img_1703670545025_105012](https://github.com/spacecowboy/Feeder/assets/147873384/f28e98b9-abf0-4393-8f64-ed4cf2fc59d9)

**Comment by spacecowboy** [comment]
Feeder's brand color is green. The specific hash can be found in `colors.xml`.

Neither of these specific instances follow material design guidelines.

The icon must also be present in other configurations so it can be re-used for other images.

AI-generated icons are not likely to be of interest.
